src/main/java/net/rpgtoolkit/common/assets/AnimationFrame.java

```diff
@@ -56,16 +56,6 @@ public AnimationFrame(String frameName, long transparentColour, String frameSoun
       } catch (IOException e) {
         e.printStackTrace();
       }
-    } else {
-      // Loads the tileset and fetches the correct tile
-      frameName = frameName.toLowerCase();

```

Comment:
Animation frames are made of images, tiles can be made of animation frames, and animations are made of animation frames. Animation frames used to be able to be made of tiles, too, but we removed that. Did I get that right?